#edit to suit your envr
image_height: 480 
image_width: 864
image_topic_out: /yolo/detect_result
image_topic_in: /camera/color/image_raw
engine_file_path: /your/file/path/yolo11x-864-480-fp32.engine
